### PR TITLE
FM2-466: Add Unit Test for Controllers in SMART-on-FHIR

### PR DESCRIPTION
**Pull request by theanandankit**
issue Link: https://issues.openmrs.org/browse/FM2-446

### CODE REVIEW

**Comment by theanandankit** [comment]
@ibacher I have added Unit tests for all Controller classes. PTAL

File: omod/src/main/java/org/openmrs/module/smartonfhir/web/filter/SmartSessionLogoutFilter.java [hunk 1]
```diff
@@ -36,13 +36,13 @@
...
-				.build(KeycloakConfigHolder.getKeycloakConfig()).getLogoutUrl();
-
+		        .build(KeycloakConfigHolder.getKeycloakConfig()).getLogoutUrl();
+		
 		if (keycloakUriBuilder != null) {
 			logoutUrl = keycloakUriBuilder.toTemplate();
 		} else {
 			log.error(
-					"Could not find Keycloak configuration file. Please run keycloak server before openmrs to avoid this error");
+			    "Could not find Keycloak configuration file. Please run keycloak server before openmrs to avoid this error");
```
Comment (HerbertYiga) [inline omod/src/main/java/org/openmrs/module/smartonfhir/web/filter/SmartSessionLogoutFilter.java:45]: do we need changes in this file?

Comment (theanandankit) [reply omod/src/main/java/org/openmrs/module/smartonfhir/web/filter/SmartSessionLogoutFilter.java:45]: yes @HerbertYiga, these changes are generated after mvn clean install.

**Review by ibacher — APPROVED** [review]
Thanks @theanandankit!

**Pull request merged**